nautobot/extras/templates/extras/jobbutton_retrieve.html

```diff
@@ -1,55 +1,2 @@
 {% extends 'generic/object_retrieve.html' %}
-{% load helpers %}
-
-{% block content_left_page %}
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <strong>Job Button</strong>
-            </div>
-
-            <table class="table table-hover panel-body attr-table">
-                <tr>
-                    <td>Object Types</td>
-                    <td>
-                        <ul>
-                            {% for ct in object.content_types.all %}
-                                <li>{{ ct }}</li>
-                            {% endfor %}
-                        </ul>
-                    </td>
-                </tr>
-                <tr>
-                    <td>Name</td>
-                    <td><span>{{ object.name }}</span></td>
-                </tr>
-                <tr>
-                    <td>Group Name</td>
-                    <td><span>{{ object.group_name }}</span></td>
-                </tr>
-                <tr>
-                    <td>Weight</td>
-                    <td><span>{{ object.weight }}</span></td>
-                </tr>
-                <tr>
-                    <td>Text</td>
-                    <td><pre>{{ object.text }}</pre></td>
-                </tr>
-                <tr>
-                    <td>Job</td>
-                    <td><a href="{% url 'extras:job_run_by_class_path' class_path=object.job.class_path %}">{{ object.job }}</a></td>
```

Comment:
Do we need a value_transform for this special case? I suspect in the current refactor this is getting rendered as a link to the Job detail view rather than the run view.